js/app.js

```diff
@@ -1,39 +1,39 @@
-const totalZombiesElement = document.querySelector('.total-zombies');
-const deadZombiesElement = document.querySelector('.dead-zombies');
-const smallZombiesElement = document.querySelector('.small-zombies');
-const madZombiesElement = document.querySelector('.mad-zombies');
-const strongZombiesElement = document.querySelector('.strong-zombies');
-
-let countTotalZombies = 0;
-let countDeadZombies = 0;
-let countSmallZombies = 0;
-let countMadZombies = 0;
-let countStrongZombies = 0;
-
-zombiesData.forEach((item) => {
-    item = Number(item);
-
-    if (item < 1 || isNaN(item)) {
-        countDeadZombies++;
-    } else if ( 0 < item && item < 11) {
-        countSmallZombies++;
-    } else if (10 < item && item < 21) {
-        countMadZombies++;
-    } else if (20 < item) {
-        countStrongZombies++;
-    }
+document.addEventListener('DOMContentLoaded', () => {
+    const zombiesWrapper = document.querySelector('.zombies');
+
+    zombies.forEach((zombie) => {
+        const zombieElement = createZombie(zombie.type);
 
-    countTotalZombies++;
+        zombiesWrapper.append(zombieElement);
+    });
 });
 
-totalZombiesElement.textContent += countTotalZombies;
-deadZombiesElement.textContent += countDeadZombies;
-smallZombiesElement.textContent += countSmallZombies;
-madZombiesElement.textContent += countMadZombies;
-strongZombiesElement.textContent += countStrongZombies;
-
-console.log(totalZombiesElement.innerText);
-console.log(deadZombiesElement.innerText);
-console.log(smallZombiesElement.innerText);
-console.log(madZombiesElement.innerText);
-console.log(strongZombiesElement.innerText);
+function createZombie(typeZombie) {
+    const zombie = document.createElement('div');
+    const imageZombie = document.createElement('img');
+
+    let image;
+
+    switch (typeZombie) {
+        case ZOMBIE_TYPE.SMALL:
+            image = './icons/zombie-small.gif';
+            break;
+        case ZOMBIE_TYPE.MAD:
+            image = './icons/zombie-mad.gif';
+            break;
+        case ZOMBIE_TYPE.STRONG:
+            image = './icons/zombie-strong.gif';
+            break;
+    }
+
+    imageZombie.setAttribute("src",image);
+    zombie.appendChild(imageZombie);
+    zombie.classList.add('zombie');
+    zombie.addEventListener('click', deadZombie);
+
+    return zombie;
+}
+
+function deadZombie(event) {
+    event.target.src = './icons/zombie-dead.gif';
```

Comment:
use an additional zombie class instead of using "img" tag and "src" attribute
just add a "dead" class on zombie element in this case